.github/workflows/workflow-java.yaml

```diff
@@ -85,30 +73,32 @@ jobs:
     uses: ./.github/workflows/run_mvn.yaml
     with:
       script: package jib:build -Djib.to.tags=${{ github.ref_name }} -Djib.target.namespace=rcsb --no-transfer-progress -Dmaven.test.skip=true -Djib.httpTimeout=0
-  # Staging jobs
-  build_jib_staging:
-    name: "Build docker image via jib with staging tag"
+  # Production jobs
+  # The production build is taken care within this job as part of rolling version in commits
+  roll_version:
     if: |
-      inputs.do_staging_build == 'true' &&
+      inputs.do_production_build == 'true' &&
       github.ref_name == inputs.mainline_branch &&
       github.event_name != 'pull_request'
+    name: "Roll maven version and release production"
     needs:
       - test_mvn
       - build_mvn
-    uses: ./.github/workflows/run_mvn.yaml
+    uses: ./.github/workflows/java-roll-version-and-release.yaml
     with:
-      script: package jib:build -Djib.to.tags=staging -Djib.target.namespace=rcsb --no-transfer-progress -Dmaven.test.skip=true -Djib.httpTimeout=0
-  # Production jobs
-  # TODO: Increment version of project through workflow. Add this step once we are ready to retire older workflow files.
-  build_jib_production:
-    name: "Build docker image via jib with production tag"
+      distribution: ${{ inputs.distribution }}
+      type: ${{ inputs.type }}
+  # Staging jobs
+  # TODO review if we actually need staging. I don't see what the use is - JD 2025-05-21
```

Comment:
And if it is wanted, we should let the calling workflow trigger it. I don't see any point to this being here while guarded by `do_staging_build`.